src/components/DurationPopover.vue

```diff
@@ -1,30 +1,62 @@
 <template>
-  <ion-select :interface-options="customPopoverOptions" interface="popover">
-    <ion-select-option value="a">Every 5 minutes</ion-select-option>
-    <ion-select-option value="b">Every 15 minutes</ion-select-option>
-    <ion-select-option value="c">Hourly</ion-select-option>
-    <ion-select-option value="d">Daily</ion-select-option>
-    <ion-select-option value="e">Disabled</ion-select-option>
+  <!-- TODO Make values dynamic and internationalise text  -->
+  <ion-select :interface-options="customPopoverOptions" interface="popover" :value="getJobStatus(id) ? getJobStatus(id) : 'SERVICE_DRAFT'" @ionChange="updateJob(id)" >
+    <ion-select-option value="EVERY_5_MIN">Every 5 minutes</ion-select-option>
+    <ion-select-option value="EVERY_15_MIN">Every 15 minutes</ion-select-option>
+    <ion-select-option value="HOURLY">Hourly</ion-select-option>
+    <ion-select-option value="DAILY">Daily</ion-select-option>
+    <ion-select-option value="SERVICE_DRAFT">Disabled</ion-select-option>
   </ion-select>
 </template>
-
 <script lang="ts">
 import { IonSelect, IonSelectOption } from '@ionic/vue';
 import { defineComponent } from 'vue';
+import { useStore } from "@/store";
+import { mapGetters } from "vuex";
 
 export default defineComponent({
   name: 'DurationPopover',
+  props: { 
+    id: String,
+    frequencyType: String
+  },
   components: {
     IonSelect,
     IonSelectOption
   },
+  computed: {
+    ...mapGetters({
+      getJobStatus: 'job/getJobStatus',
+      getJob: 'job/getJob'
+    })
+  },
+  methods: {
+    async updateJob(id: string) {
```

Comment:
I think we haven't handled the case when user disabled the job. Let me know if I missed something